PFire.sln

```diff
@@ -7,7 +7,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PFire.Console", "src\PFire.
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PFire.Core", "src\PFire.Core\PFire.Core.csproj", "{62FD083C-F667-44FA-99E7-7A35F363886B}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PFire.Infrastructure", "src\PFire.Infrastructure\PFire.Infrastructure.csproj", "{CB7EC8E0-436A-4E63-9F24-020238058DB2}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PFire.Data", "src\PFire.Data\PFire.Data.csproj", "{CB7EC8E0-436A-4E63-9F24-020238058DB2}"
```

Comment:
I prefer Infrastructure and have database part of that infrastructure. If we add any other external-to-the-domain services, they'd live in this project too and sometimes aren't necessarly data related.

Not a big deal at the moment though.

Reply:
Ok, not a problem! I'll put it back